### PR TITLE
fix: display progress bar in product's card

### PR DESCRIPTION
**Pull request by Ahmed-S-Abduallah**
What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)

* bug fix

What is the current behaviour? (You can also link to an open issue here)

* The progress bar is not visible

What is the new behaviour? (You can also link to the ticket here)

* The progress bar is visible

Does this PR introduce a breaking change?

* no

Screenshots (If appropriate)

*

### CODE REVIEW

File: src/assets/js/partials/product-card.js [hunk 1]
```diff
@@ -231,7 +231,7 @@ class ProductCard extends HTMLElement {
               : ``}
           </div>
           ${this.product?.donation && !this.minimal && !this.fullImage ?
-          `<salla-progress-bar donation=${this.product?.donation} />
+          `<salla-progress-bar donation=${JSON.stringify(this.product.donation)} /></salla-progress-bar>
```
Comment (jalmatari) [inline src/assets/js/partials/product-card.js:234]: remove `</salla-progress-bar>`
`/>` should do the same, or remove `/` from `/>`

**Review by jalmatari — CHANGES_REQUESTED** [review]

**Review by Ahmed-S-Abduallah — COMMENTED** [review]
fixed

**Review by Ahmed-S-Abduallah — COMMENTED** [review]
fixed

**Review by jalmatari — APPROVED** [review]

**Comment by SallaDev** [comment]
https://github.com/SallaApp/theme-raed/pull/392 is been merged ✅

-------------------

Merge by: @jalmatari

**Pull request merged**